pkg/controllers/allocation/binpacking/packable.go

```diff
@@ -104,6 +109,20 @@ func (p *Packable) Pack(pods []*v1.Pod) *Result {
 	return result
 }
 
+// isFull checks if the reserved resources + the minimum resourced pod overflows the total resources available.
+func (p *Packable) isFull(minPod *v1.Pod) bool {
```

Comment:
optional: s/minPod/pod : you could imagine that we'd check pods that we're min if we decided to not sort for some reason.
optional: s/isFull/fits : isFull seems a bit odd since the node is only full in relation to the pod. The node isn't necessarily full, since another pod might fit if it had no resource reqs.